### PR TITLE
Allow puppetlabs/concat 6.x, puppetlabs/stdlib 6.x, puppet/nodejs 7.x; require at least stdlib 4.25.0 instead of 4.17

### PR DESCRIPTION
**Pull request by dhoppe**

### CODE REVIEW

**Comment by dhoppe** [comment]
I think this might require a new release of puppet/nodejs.

**Comment by dhoppe** [comment]
@bastelfreak The build passes, finally.

File: metadata.json [hunk 1]
```diff
@@ -8,21 +8,21 @@
...
   "issues_url": "https://github.com/voxpupuli/puppet-etherpad/issues",
   "dependencies": [
+    {
+      "name": "puppetlabs-concat",
+      "version_requirement": ">= 4.7.0 < 7.0.0"
+    },
     {
       "name": "puppetlabs-stdlib",
-      "version_requirement": ">= 4.17.0 < 6.0.0"
+      "version_requirement": ">= 4.25.0 < 7.0.0"
```
Comment (Dan33l) [inline metadata.json]: if we bump the minimal version required, the PR needs to be marked as `backwards-incompatible`

**Comment by bastelfreak** [comment]
marked as bug because of https://github.com/voxpupuli/puppet-openvpn/pull/340

**Pull request merged**